evelien-kortrijk.md

```diff
@@ -0,0 +1,19 @@
+# Kortrijk,Belgium
+
+The city is on the river Leie, 42 km (26 mi) southwest of Ghent and 25 km (16 mi) northeast of Lille
+
+## Restaurants
+
+Plenty of places to eat
```

Comment:
Could you give me some examples? I like anything, but I ❤️ 🍣.

Reply:
Sakura Japanese restaurant